dm/ctl/common/util.go

```diff
@@ -19,11 +19,17 @@ import (
 	"fmt"
 	"io/ioutil"
 	"reflect"
+	"regexp"
+	"strconv"
 	"strings"
 	"sync"
 	"time"
 
+	"github.com/pingcap/failpoint"
 	"go.etcd.io/etcd/clientv3"
+	"go.uber.org/zap"
+
+	"github.com/pingcap/dm/pkg/log"
 
 	"github.com/pingcap/dm/dm/config"
 	"github.com/pingcap/dm/dm/pb"
```

Comment:
fmt

Reply:
I think this is caued by our `make fmt` @Ehco1996 PTAL

Reply:
our fmt tools just can't group this imports correctly... but you can formart manually  - -.

Reply:
Can you fix these imports manually?